instrumentation/elastic-opentelemetry-instrumentation-openai/tests/test_beta_chat_completions.py

```diff
@@ -1509,6 +1510,159 @@ def test_chat_exported_schema_version(default_openai_env, trace_exporter, metric
             assert scope_metrics.schema_url == "https://opentelemetry.io/schemas/1.28.0"
 
 
+@pytest.mark.skipif(OPENAI_VERSION < (1, 40, 0), reason="beta completions added in 1.40.0")
+@pytest.mark.vcr()
+def test_parse_response_format_json_object_with_capture_message_content(
+    default_openai_env, trace_exporter, metrics_reader, logs_exporter
+):
+    # Redo the instrumentation dance to be affected by the environment variable
+    OpenAIInstrumentor().uninstrument()
+    with mock.patch.dict("os.environ", {"OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT": "true"}):
+        OpenAIInstrumentor().instrument()
+
+    client = openai.OpenAI()
+
+    from pydantic import BaseModel
+
+    class Reason(BaseModel):
+        reason: str
+
```

Comment:
It looks like you are not using these here

Reply:
Good catch.